pbrEngine/engine.py

```diff
@@ -691,6 +749,291 @@ def _resetBlur(self):
+
+    def _enableStargazerMusic(self):
+        """
+        Enables the music for the Stargazer Colosseum. (Battletopia06)
+        """
+        self._enableSong(Locations.SONG_STARGAZER, "sound/Battletopia06.brstm")
+
+    def _enableGatewayMusic(self):
+        """
+        Enables the music for the Gateway Colosseum. (Battletopia07)
+        """
+        self._enableSong(Locations.SONG_GATEWAY, "sound/Battletopia07.brstm")
+
+    def _enableNeonMusic(self):
+        """
+        Enables the music for the Neon Colosseum. (Battletopia08)
+        """
+        self._enableSong(Locations.SONG_NEON, "sound/Battletopia08.brstm")
+
+    def _enableMainStreetMusic(self):
+        """
+        Enables the music for the Main Street Colosseum. (Battletopia09)
+        """
+        self._enableSong(Locations.SONG_MAIN_STREET, "sound/Battletopia09.brstm")
+
+    def _enableCrystalMusic(self):
+        """
+        Enables the music for the Crystal Colosseum. (Battletopia10)
+        """
+        self._enableSong(Locations.SONG_CRYSTAL, "sound/Battletopia10.brstm")
+
+    def _enableLagoonMusic(self):
+        """
+        Enables the music for the Lagoon Colosseum.
+        """
+        self._enableSong(Locations.SONG_LAGOON, "sound/pokerev_music_demo_v2.brstm")
+
+    def _enableFanfareVar1(self):
+        """
+        Enables the first variation of the fanfare. (ME_Fan01)
+        """
+        self._enableSong(Locations.SONG_FANFARE_VAR1, "sound/ME_Fan01.brstm")
+
+    def _enableFanfareVar3(self):
+        """
+        Enables the third variation of the fanfare. (ME_Fan03)
+        """
+        self._enableSong(Locations.SONG_FANFARE_VAR3, "sound/ME_Fan03.brstm")
+
+    def _enableFanfareVar5(self):
+        """
+        Enables the fifth variation of the fanfare. (ME_Fan05)
+        """
+        self._enableSong(Locations.SONG_FANFARE_VAR5, "sound/ME_Fan05.brstm")
+
+    def _enableFanfareCompleted(self):
+        """
+        Enables the fanfare that plays when you have completed a colosseum. (ME_Fin)
+        """
+        self._enableSong(Locations.SONG_FANFARE_COMPLETED, "sound/ME_Fin.brstm")
+
+    def _enableColosseumSelectionMusic(self):
+        """
+        Enables the music for the Colosseum Selection screen. (System01)
+        """
+        self._enableSong(Locations.SONG_COLOSSEUM_SELECTION, "sound/System01.brstm")
+
+    def _enableReceptionMusic(self):
+        """
+        Enables the music for the Reception screen. (System03)
+        """
+        self._enableSong(Locations.SONG_RECEPTION, "sound/System03.brstm")
+
+    def _enableMonSelectionMusic(self):
+        """
+        Enables the music for the Pokemon Selection screen. (System05)
+        """
+        self._enableSong(Locations.SONG_MON_SELECTION, "sound/System05.brstm")
+
+    def _enableControlsMusic(self):
+        """
+        Enables the music for the Controls screen. (System09)
+        """
+        self._enableSong(Locations.SONG_CONTROLS, "sound/System09.brstm")
```

Comment:
do these need to have a separate method each? I suggest just putting them in a list, maybe tuples with their respective defaults